administrator/components/com_content/tmpl/article/edit.php

```diff
@@ -104,6 +127,79 @@
 			<?php echo HTMLHelper::_('uitab.endTab'); ?>
 		<?php endif; ?>
 
+		<?php // Do not show the publishing options if the edit form is configured not to. ?>
+		<?php if ($params->get('show_imported_modules', 1) == 1 && \count($this->item->importedModules)): ?>
```

Comment:
Notice and Warning:  warning: count(): Parameter must be an array or an object that implements Countable

Use if (!... empty($this->item->importedModules))

Reply:
Thank you!